.eslintignore

```diff
@@ -3,4 +3,5 @@ example
 dist
 lib
 packages/**/src/types.tsx
+packages/**/src/foundation.tsx
```

Comment:
why do you need to remove this from lint checks ?

Reply:
That is because of the arguments type mismatch: we swapped `timestamp` and `context` in the interface definition, but they remain at the old positions at the bridge side. So TypeScript complains about type mismatch. To fix that it is possible to add `\\ @ts-ignore` comment, to suppress error, but then linter complains about the usage of `@ts-ignore`, so adding this file to ignore-list for now.

Once we update bridges, this won't be needed.

Reply:
So maybe add a todo there to not forget to remove it when you update the bridges. Other than that this look very well done to me.